src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -223,6 +223,7 @@
     <Compile Include="ViewModels\Core\DynamoViewModelBranding.cs" />
     <Compile Include="ViewModels\Core\GalleryViewModel.cs" />
     <Compile Include="ViewModels\Core\HomeWorkspaceViewModel.cs" />
+    <Compile Include="ViewModels\Core\PackagePathViewModel.cs" />
```

Comment:
I think this belongs to the new project `DynamoPublish` because package manager related stuff has been moved out of the core.